bin/bundle

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env ruby
```

Comment:
Don't need any of the files in the `bin/` directory. Please remove these files too.